# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1368602254 14400
#      Wed May 15 03:17:34 2013 -0400
# Node ID af7287d567dc6fe933efa62d6095a0855e6643a5
# Parent  d68e72ee28bf6ef4e582a651554cf877226b2514
minor change to wording for previous change

diff --git a/doc/interpreter/contrib.txi b/doc/interpreter/contrib.txi
--- a/doc/interpreter/contrib.txi
+++ b/doc/interpreter/contrib.txi
@@ -43,18 +43,19 @@ You may consider developing and publishi
 place for this is the allied Octave-Forge project
 (@url{http://octave.sourceforge.net}).  Note that the Octave project is
 inherently more conservative and follows narrower rules.
 
 @node Building the Development Sources
 @section Building the Development Sources
 
 The directions for building from the Development sources change from
-time to time, so you should read the resources for developers.  Start
-here: @url{http://www.octave.org/get-involved.html}.
+time to time, so you should read the resources for developers on the web
+or in the development sources archive.  Start here:
+@url{http://www.octave.org/get-involved.html}.
 
 @node Basics of Generating a Changeset
 @section Basics of Generating a Changeset
 
 The preferable form of contribution is creating a Mercurial changeset
 and submit it to the @uref{http://savannah.gnu.org/bugs/?group=octave, bug} or
 @uref{http://savannah.gnu.org/patch/?func=additem&group=octave, patch}
 trackers@footnote{Please use the patch tracker only for patches which add new
